code/modules/events/meteor_wave.dm

```diff
@@ -59,11 +61,21 @@
 			kill()
 
 /datum/round_event/meteor_wave/announce(fake)
-	priority_announce("Meteors have been detected on collision course with the station. Estimated time until impact: [round(startWhen/60)] minutes.[GLOB.singularity_counter ? " Warning: Anomalous gravity pulse detected, Syndicate technology interference likely." : ""]", "Meteor Alert", "meteors")
+	var/directionstring
+	switch(direction)
+		if(NORTH)
+			directionstring = " from the north."
+		if(SOUTH)
+			directionstring = " from the south."
+		if(EAST)
+			directionstring = " from the east."
+		if(WEST)
+			directionstring = " from the west."
```

Comment:
Nautical terms (such as fore, aft, startboard, port) please, it is pretty wrong to use cardinal directions on automated announcements and the such when there is no north star to orientate around in the game. (Yes, we'll still gergally use cardinals, but think of this as an incentive to know these basic terms)

Also should replace "from" with "toward" since the terms refer to sides of the ship and not the direction.